src/styles/card.scss

```
@@ -0,0 +1,115 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 200px;
+  padding: 32px 16px 16px;
+  background-color: $background-color;
+  border-radius: 5px;
+  border: 1px solid $border-color;
+  transition: transform 300ms;
```

you repeat it 5 times, so, put it to the variable)
```suggestion
  transition: transform ...;
```